tutorial_2.md

```diff
@@ -1,41 +1,69 @@
+The automation scripts in this tutorial use the LaunchDarkly REST API to programmatically create configurations. Here's how to get your API key:
 
-**Finding MCP Servers:** The [MCP Registry](https://github.com/modelcontextprotocol/registry) serves as a community-driven directory for discovering available MCP servers - like an "app store" for MCP tools. Browse available servers at [registry.modelcontextprotocol.io](https://registry.modelcontextprotocol.io/docs#/operations/list-servers) to see what's currently available. Registry servers install through standard package managers:
+To get your LaunchDarkly API key, start by navigating to Organization Settings by clicking the gear icon (⚙️) in the left sidebar of your LaunchDarkly dashboard. Once there, access Authorization Settings by clicking **"Authorization"** in the settings menu. Next, create a new access token by clicking **"Create token"** in the "Access tokens" section.
+
+![API Token Creation](screenshots/api_token.png)
+*Click "Create token" in the Access tokens section*
+
+When configuring your token, give it a descriptive name like "multi-agent-chatbot", select **"Writer"** as the role (required for creating configurations), use the default API version (latest), and leave "This is a service token" unchecked for now.
+
+![Name API Token](screenshots/name_api_token.png)
+*Configure your token with a descriptive name and Writer role*
+
+After configuring the settings, click **"Save token"** and immediately copy the token value. This is **IMPORTANT** because it's only shown once!
+
+![Copy API Token](screenshots/copy_api_token.png)
+*Copy the token value immediately - it's only shown once*
+
+Finally, add the token to your environment:
+   ```bash
+   # Add this line to your .env file
+   LD_API_KEY=your-copied-api-key-here
+   ```
+
+**Security Note**: Keep your API key private and never commit it to version control. The token allows full access to your LaunchDarkly account.
+
+## Step 1: Add External Research Tools (4 minutes)
+
+Your agents need more than just your internal documents. **Model Context Protocol (MCP)** connects AI assistants to live external data and they agents become orchestrators of your digital infrastructure, tapping into databases, communication tools, development platforms, and any system that matters to your business. MCP tools run as separate servers that your agents call when needed.
+
+The [MCP Registry](https://github.com/modelcontextprotocol/registry) serves as a community-driven directory for discovering available MCP servers - like an app store for MCP tools. Browse available servers at [registry.modelcontextprotocol.io](https://registry.modelcontextprotocol.io/docs#/operations/list-servers) to see what's currently available. Registry servers install through standard package managers:
 
 ```bash
 # Example registry installations (not used in this tutorial)
```

Comment:
This section is a little confusing. I think it's a little unnecessary to add installation instructions for MCP servers that aren't used in this tutorial.